### ISSUE TITLE
是否有计划支持qwen-image的dpo训练？

### ISSUE THREAD

**Issue by vibe-viscot** [modelscope/DiffSynth-Studio#770]
dpo的具体细节可见qwen-image的报告：https://arxiv.org/abs/2508.02324

**Comment by lifan724** [comment]
尝试改了下，放弃了，几个问题：
1. diffsyth 现在都是bs=1 训练， dpo win&loss bs=2，数据读取逻辑和推理过程得改
2. 现有diffsyth pip.dit 不支持diffusers那样通过 lora-adapters进行 new/old policy加卸载, 得重写dit模块
3. 存两个dit，但是qwen-image20B dit存两个太难了，不开zero3, 140G的卡都撑不住
4. 维持bs=1, forward 两次，但是现在框架把loss.backward()和梯度累积的逻辑抽象出了公共父类，此路不大改，也不通 :( sad
希望官方下一阶段diffsyth的代码不要过度抽象，现在写的太散，改rl-loss很麻烦
